test/randomized/test_clifford.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Randomized tests of Clifford operator class."""
+import unittest
+from hypothesis import given, strategies
+
+from qiskit.quantum_info.random import random_clifford
+from qiskit.quantum_info import Clifford
+
+
+class TestClifford(unittest.TestCase):
+    """Test random_clifford"""
+
+    def assertValidClifford(self, value, num_qubits):
+        """Assertion from test/python/quantum_info/operators/test_random.py:
+        TestRandomClifford:test_valid"""
+        with self.subTest(msg='Test type'):
```

Comment:
Looks like there's an error setting up the subTest context manager? https://travis-ci.com/github/Qiskit/qiskit-terra/jobs/321357365#L1038